executor/include/execution/trait/is_executor.hpp

```diff
@@ -21,11 +21,13 @@ template <typename Executor, typename Function>
 struct is_executor<
     Executor, Function,
     std::enable_if_t<
-        std::is_copy_constructible_v<
-            std::remove_cv_t<std::remove_reference_t<Executor>>> &&
-            std::is_same_v<
-                decltype(std::declval<Executor>().execute(detail::noop)), void>,
-        decltype(std::declval<Executor&>() == std::declval<Executor&>(),
+        std::is_copy_constructible<
+            std::remove_cv_t<std::remove_reference_t<Executor>>>::value &&
+            std::is_same<decltype(std::declval<std::remove_cv_t<
+                                      std::remove_reference_t<Executor>>>()
+                                      .execute(detail::noop)),
+                         void>::value,
+        decltype(std::declval<Executor>() == std::declval<Executor>(),
                  (void)0)>> : std::true_type {};
```

Comment:
I'm still curious about this syntax, since you're apparently passing to argument to `decltype` which just takes one. Can you provide details about it?